README.md

```diff
@@ -358,7 +362,7 @@ async function main() {
 main().catch(console.error);
 ```
 
-A complete example can be found in [`samples/idtokens-cloudrun.js`](https://github.com/googleapis/google-auth-library-nodejs/blob/master/samples/idtokens-cloudrun.js).
+A complete example can be found in [`samples/idtokens-serverless.js`](https://github.com/googleapis/google-auth-library-nodejs/blob/master/samples/idtokens-serverless.js).
```

Comment:
Just double checking - have you done a check on devrel/snippets to see if we need to update paths anywhere?

Reply:
Does not appear in devrel/snippets.